src/schemas/json/sarif-2.0.0.json

```diff
@@ -0,0 +1,1954 @@
+
+        "instanceGuid": {
+          "description": "A stable, unique identifer for the result in the form of a GUID.",
+          "type": "string"
+        },
+
+        "partialFingerprints": {
+          "description": "A set of strings that contribute to the stable, unique identity of the result.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+
+        "fingerprints": {
+          "description": "A set of strings each of which individually defines a stable, unique identity for the result.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+
+        "stacks": {
+          "description": "An array of 'stack' objects relevant to the result.",
+          "type": "array",
+          "minItems": 1,
+          "uniqueItems": true,
+          "items": {
+            "$ref": "#/definitions/stack"
+          }
+        },
+
+        "codeFlows": {
+          "description": "An array of 'codeFlow' objects relevant to the result.",
+          "type": "array",
+          "minItems": 1,
+          "uniqueItems": true,
+          "items": {
+            "$ref": "#/definitions/codeFlow"
+          }
+        },
+
+        "graphs": {
+          "description": "An array of one or more unique 'graph' objects.",
+          "type": "array",
+          "minItems": 1,
+          "uniqueItems": true,
+          "items": {
+            "$ref": "#/definitions/graph"
+          }
+        },
+
+        "graphTraversals": {
+          "description": "An array of one or more unique 'graphTraversal' objects.",
+          "type": "array",
+          "minItems": 1,
+          "uniqueItems": true,
+          "items": {
+            "$ref": "#/definitions/graphTraversal"
+          }
+        },
+
+        "relatedLocations": {
+          "description": "A set of locations relevant to this result.",
+          "type": "array",
+          "minItems": 1,
+          "uniqueItems": true,
+          "items": {
+            "$ref": "#/definitions/location"
+          }
+        },
+
+        "suppressionStates": {
+          "type": "array",
+          "items": {
+            "description": "A flag value indicating one or more suppression conditions.",
+            "enum": [
+              "suppressedInSource",
+              "suppressedExternally"
+            ]
+          }
+        },
+
+        "baselineState": {
+          "description": "The state of a result relative to a baseline of a previous run.",
+          "enum": [
+            "new",
+            "existing",
+            "absent"
+          ]
+        },
+
+        "attachments": {
+          "description": "A set of files relevant to the result.",
+          "type": "array",
+          "minItems": 1,
+          "uniqueItems": true,
+          "items": {
+            "$ref": "#/definitions/attachment"
+          }
+        },
+
+        "workItemLocation": {
```

Comment:
@michaelcfanning @EasyRhino0

Michael, this isn't the final version. I can tell because it should be `workItemUris`, an array of URI-format strings.

As of Chris's most recent work, this property is `workItemUri`, a single URI-format string. I'm investigating a codegen crash that happened when he tried to make it into an array.